test/test_plugin.rb

```diff
@@ -28,6 +28,23 @@ def test_plugin
     out.close
   end
 
+  def test_non_bundled_plugin
+    skip "Skipped on Windows Ruby < 2.5.0, Ruby bug" if windows? && RUBY_VERSION < '2.5.0'
```

Comment:
I assume this is just copied from above, but I wonder if it is still true for the test below?

Reply:
yeah try leaving that skip out

Reply:
Done

Reply:
Looks like that's a "no" lol https://github.com/puma/puma/pull/2032/checks?check_run_id=260440680#step:9:121

Reply:
Good to know... I wonder what bug this is, I guess @MSP-Greg must have added this skip initially

Reply:
lol -- reverted, btw I have a windows machine somewhere, I would test this out on windows but it hurts so much using windows 😂

Reply:
Cute.  I'm 60+, I can say that to someone your age.  Anyway, Windows should work now.  BTW, running from just rake needs a `pathname` require.

Reply:
Nice work on that PR @MSP-Greg I can see that windows works 👍✨